src/docs/asciidoc/reference_doc/response_cards.adoc

```diff
@@ -75,31 +76,48 @@ A card can have a response only if it's in a process/state that is configured fo
 }
 ....
 
-We define here a state name "questionState" with a response field. Now, if we send a card with process "defaultProcess" and state "questionState" , the user will have the possibility to respond if he has the good privileges. 
+We define here a state name "questionState" with a response field. Now, if we send a card with process "defaultProcess"
+and state "questionState", the user will have the possibility to respond if he has the required privileges.
 
 - The field "state" in the response field is used to define the state to use for the response (the child card).
-- The field "externalRecipients" define the recipients of the response card. These recipients are keys referenced in the config file of cards-publication service, in "externalRecipients-url" element. This field is optional.
-- The field "btnColor" define the color of the submit button for the response, it is optional and there is 3 possibilities : RED , GREEN , YELLOW 
-- The field "btnText"is the i18n key of the title of the submit button, it is optional.
+- The field "externalRecipients" define the recipients of the response card. These recipients are keys referenced in
+the config file of cards-publication service, in "externalRecipients-url" element. This field is optional.
+- The field "btnColor" define the color of the submit button for the response, it is optional and there are 3
+possibilities : RED , GREEN , YELLOW
+- The field "btnText" is the i18n key of the title of the submit button, it is optional.
 
+NOTE: The state to be used for the response can also be set dynamically based on the contents of the card or the
+response by returning it in the `templateGateway.validyForm` method (see below for details).
 
 === Design the question form in the template
 
 For the user to response you need to define the response form in the template with standard HTML syntax   
 
-To enable operator fabric to send the response, you need to implement a javascript function in your template called templateGateway.validyForm which return an object containing three fields :
+To enable operator fabric to send the response, you need to implement a javascript function in your template called
+`templateGateway.validyForm` which returns an object containing four fields :
 
 - valid (_boolean_) : true if the user input is valid
 - errorMsg (_string_) : message in case of invalid user input. If valid is true this field is not necessary.
 - formData (_any_) : the user input to send in the data field of the child card. If valid is false this field is not necessary.
+- responseState : name of the response state to use.
```

Comment:
precise that it is not mandatory , the default value being the one in config.json

Reply:
Done thanks!